cmd/restic/global.go

```diff
@@ -376,6 +376,10 @@ func OpenRepository(opts GlobalOptions) (*repository.Repository, error) {
 		return s, nil
 	}
 
+	if c.Created {
+		Verbosef("created new cache in %v\n", c.Base)
```

Comment:
Will this print only when the `--json` flag is _not_ used? (Printing non-JSON breaks JSON parsers)

Reply:
Uh, good point, thanks!

Reply:
At some point I'd like to refactor how messages are printed in restic, so we can handle such things in a more general way...

Reply:
Fixed it :)